net/ntopng/src/opnsense/mvc/app/models/OPNsense/Ntopng/General.xml

```diff
@@ -0,0 +1,30 @@
+<model>
+    <mount>//OPNsense/ntopng/general</mount>
+    <description>ntopng configuration</description>
+    <version>0.0.1</version>
+    <items>
+        <enabled type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+        </enabled>
+        <interface type="InterfaceField">
+            <Required>N</Required>
+            <Multiple>N</Multiple>
+        </interface>
+        <httpport type="IntegerField">
+            <Required>Y</Required>
+            <default>3000</default>
+            <MinimumValue>0</MinimumValue>
```

Comment:
I am not sure if this makes sense - in the bsd socket API 0 means random port - might be a different meaning in this software.

Reply:
0 means disabled. Normally I would use PortField, but it doesn't allow usage of 0.

Reply:
But if you set Required N in the PortField you can leave it blank and use it as disabled? Passing "0" that doesn't work as a port to do something else is worse than adding a checkbox or other means for disabling like empty field.

Reply:
I removed the 0 thing .. it doesn't make sense since the service stops as a whole when the port is set to 0 :)